ecosystem/indexer-grpc/indexer-grpc-cache-worker/src/worker.rs

```diff
@@ -291,12 +328,14 @@ async fn process_streaming_response(
         if received.chain_id as u64 != fullnode_chain_id as u64 {
             panic!("[Indexer Cache] Chain id mismatch happens during data streaming.");
         }
-
+        let size_in_bytes = received.encoded_len();
```

Comment:
This is the size in bytes of the TransactionsFromNodeResponse

Reply:
done. used the accumulated size to reflect the batch size.

Reply:
ok finished with new variable `total_size_in_bytes`

Reply:
We should split them up into different events but keep the same variable name `size_in_bytes` so we can plot them in the same chart.